### PR TITLE
creating governance folder + adding coworking & core team notes

### LINKED ISSUE #3200: Upload coworking call notes
Adding meeting notes from core team:
- [ ] Create new 'governance' folder
- [ ] Add notes from core team meeting (9 June)
- [ ] Upload notes for 5 & 13 June coworking calls

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Will close #3200 

Adds notes from two coworking calls in june (5 & 12 june) and core team meeting (9 june)

### What I need help with
- What do we think of this file structure? (especially for the coworking calls)
- Are there any other initial notes missing from this collection? (i.e. collaboration cafe)
- Anything else missing from this PR?

### CODE REVIEW

**Review by AlexandraAAJ — COMMENTED** [review]
All good from my side.

**Review by Arielle-Bennett — APPROVED** [review]
one minor update but otherwise all good!

File: governance/core-team/meeting-notes/20230609-core-team.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+https://turing-uk.zoom.us/j/92314453636?pwd=alU5MnJuT1NIZ3kzT3RnUk1lQmRjUT09
+
+Meeting ID: 923 1445 3636
+Passcode: 871311
+:::
+
+### Code of conduct
+
+_The Turing Way_ Code of conduct applies to this call.
+* [Take a moment to read this](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE_OF_CONDUCT.md)
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/core-team/meeting-notes/20230609-core-team.md]: Thank you @Arielle-Bennett!

**Comment by aleesteele** [comment]
Thank you also @AlexandraAAJ!

File: governance/community-calls/20230605-coworking.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    * We discussed where we were with our project board tasks: https://github.com/orgs/alan-turing-institute/projects/32/views/1
+    * Decided to ask KW, MS about onboarding tasks
+    * Decided to start work on training workshop templates and new chapter about this - EK to start
+    * Alden will update the list of presentations and Eirini will help to check this.
+* Batool: quiet room
+    * I worked on RSF-related tasks (forcasts, emails, docs)
+* Main room: Governance
+  * Pom 1: Kirstie, Bastian, Cami and Jessie
+    * :zipper_mouth_face: State of governance document (not for github)
+      * https://docs.google.com/document/d/1Z_r0G6dmtlw0icQ_ctkhLG_QxZDhuRZaSTSkGvAVK6w/edit
```
Comment (KirstieJane) [inline governance/community-calls/20230605-coworking.md:54]: Remove link - not for GitHub

File: governance/community-calls/20230605-coworking.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+* Alden, Eirini, Emma: trainers and mentors
+    * We discussed where we were with our project board tasks: https://github.com/orgs/alan-turing-institute/projects/32/views/1
+    * Decided to ask KW, MS about onboarding tasks
+    * Decided to start work on training workshop templates and new chapter about this - EK to start
+    * Alden will update the list of presentations and Eirini will help to check this.
+* Batool: quiet room
+    * I worked on RSF-related tasks (forcasts, emails, docs)
+* Main room: Governance
+  * Pom 1: Kirstie, Bastian, Cami and Jessie
+    * :zipper_mouth_face: State of governance document (not for github)
```
Comment (KirstieJane) [inline governance/community-calls/20230605-coworking.md:53]: Remove row - document not for GitHub

File: governance/core-team/meeting-notes/20230609-core-team.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+Note of notes for Core/Organisational Team: https://hackmd.io/@turingway/ttw-team-note-of-notes
+:::
+
+Meeting information:
+:::info
+https://turing-uk.zoom.us/j/92314453636?pwd=alU5MnJuT1NIZ3kzT3RnUk1lQmRjUT09
+
+Meeting ID: 923 1445 3636
+Passcode: 871311
+:::
```
Comment (KirstieJane) [inline governance/core-team/meeting-notes/20230609-core-team.md:13]: Remove? Doesn't need to be in GitHub archive and makes zoom bombing too easy

File: governance/core-team/meeting-notes/20230609-core-team.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+| -------- | -------- | -------- |
+| **Frameworks** | **Strategy** | **Culture**
+| - Guidelines for WG to make decisions (Decision trees) |- Long-term Structures | - Differences & clarity about paid and volunteers team members
+| - Guidelines to onboard members and structure for growth (Mountain of Engagement)     | - Short-term Structure | - TTW Funding approaches (paying non-Turing members)
+| - Guidelines for engagement (Personas & Pathways)  | - Separating strategic & operational responsibilities  | - Who has the power to make decisions?
+
+Therefore, we would like to use this meeting to:
+
+- Share the Governance documentation documents
+    - 🤫 Draft 1: https://docs.google.com/document/d/1Z_r0G6dmtlw0icQ_ctkhLG_QxZDhuRZaSTSkGvAVK6w/edit#heading=h.tpy4degl3nyk
```
Comment (KirstieJane) [inline governance/core-team/meeting-notes/20230609-core-team.md:81]: Remove - not for GitHub

File: governance/core-team/meeting-notes/20230609-core-team.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+| **Frameworks** | **Strategy** | **Culture**
+| - Guidelines for WG to make decisions (Decision trees) |- Long-term Structures | - Differences & clarity about paid and volunteers team members
+| - Guidelines to onboard members and structure for growth (Mountain of Engagement)     | - Short-term Structure | - TTW Funding approaches (paying non-Turing members)
+| - Guidelines for engagement (Personas & Pathways)  | - Separating strategic & operational responsibilities  | - Who has the power to make decisions?
+
+Therefore, we would like to use this meeting to:
+
+- Share the Governance documentation documents
+    - 🤫 Draft 1: https://docs.google.com/document/d/1Z_r0G6dmtlw0icQ_ctkhLG_QxZDhuRZaSTSkGvAVK6w/edit#heading=h.tpy4degl3nyk 
+    - 🤫 Draft 2: https://docs.google.com/document/d/18DvwnQ88Bwo_5QVHNdGBrKf2gQH1-1wqzTrDvGSQ874/edit#heading=h.ffcz13kp42oa
```
Comment (KirstieJane) [inline governance/core-team/meeting-notes/20230609-core-team.md:82]: Remove - not for GitHub

File: governance/core-team/meeting-notes/20230609-core-team.md [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+* Identify the sections/headings our Governance documents will
+* Develop a timeline and consult each section with community members
+* **Urgent need**: Produce a governance official document  
+
+
+## Guided discussion for core team (30 min)
+
+[ Kirstie ]
+
+Building on the 1-3-3 all Librating Format (https://www.liberatingstructures.com/1-1-2-4-all/), we will be using this time to learn more about what teams need.
```
Comment (KirstieJane) [inline governance/core-team/meeting-notes/20230609-core-team.md:100]:
```suggestion
Building on the 1-2-4-all Librating Format (https://www.liberatingstructures.com/1-1-2-4-all/), we will be using this time to learn more about what teams need.
```

**Review by KirstieJane — COMMENTED** [review]
A few suggestions of sensitive links that should be removed. Otherwise good from my side!

**Comment by KirstieJane** [comment]
Oh whoops - I didn't see on my phone that this was already merged.

@aleesteele - I'll ping you on slack to please open a new PR to remove the sensitive links.

@AlexandraAAJ and @Arielle-Bennett - can you explain why you missed the 🤫 and 🤐 "not for GitHub" lines? Am I missing something?

**Pull request merged**